src/main/scala/iog/psg/cardano/ApiRequestExecutor.scala

```diff
@@ -0,0 +1,18 @@
+package iog.psg.cardano
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import iog.psg.cardano.CardanoApi.{CardanoApiRequest, CardanoApiResponse}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class ApiRequestExecutorImpl extends ApiRequestExecutor
```

Comment:
actually I am considering here, maybe just leave a class, without this trait, wdyt ?
will be simpler